src/components/layout/dashboard/contextHeader/ContextHeader.tsx

```diff
@@ -115,7 +117,7 @@ const ContextHeader: React.FC<Props> = ({
         </Box>
       ) : (
         <Box display='flex' alignItems='stretch' width='100%' flex={1}>
-          {!isOpen && (
+          {!isOpen && !isMobile && (
```

Comment:
Prevent both menu items from  appearing. To reproduce, make sure menu is closed when you reduce screen width
<img width="746" alt="Screenshot 2024-08-13 at 4 54 00 PM" src="https://github.com/user-attachments/assets/e0efbcfb-4fa7-4a1a-8428-eb4d8b7ba161">